ci/check_style.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
@ajschmidt8 This file can be dramatically simplified now that #420 has been merged.